apps/data_handler/handler_tools/constants.py

```diff
@@ -6,6 +8,7 @@
 # Temporary solution.
 # TODO remove it when all protocols with interest rate models will be available
 class AvailableProtocolID(Enum):
+    """class docstring"""
```

Comment:
It won't work this way, it should contain description of the class not just some random text